docs/existing-aws-infrastructure.md

```diff
@@ -7,45 +7,27 @@ Normally, Cluster API will create infrastructure on AWS when standing up a new w
+* One or more private subnets (subnets that do not have a route to an Internet gateway)
 * A public subnet in the same Availability Zone (AZ) for each private subnet (this is required for NAT gateways to function properly)
 * A NAT gateway for each private subnet, along with associated Elastic IP addresses
 * An Internet gateway for all public subnets
 * Route table associations that provide connectivity to the Internet through a NAT gateway (for private subnets) or the Internet gateway (for public subnets)
 
-You will need the following information, which can be obtained either via the AWS Management Console or using the AWS CLI:
+Note that a public subnet (and associated Internet gateway) are required for the control plane of the workload cluster.
 
-1. The ID of the VPC
-2. The IDs of the subnets in the VPC that you want Cluster API to use
+You will need the ID of the VPC that Cluster API should use. This information is available via the AWS Management Console or the AWS CLI. It is not necessary to have a list of the subnet IDs; Cluster API will discover the subnets if the VPC ID is provided.
 
 Note that there is no need to create an Elastic Load Balancer (ELB), security groups, or EC2 instances; Cluster API will take care of these items.
 
 ## Tagging AWS Resources
 
-In order for Cluster API to properly recognize and consume existing AWS resources, these resources must be tagged with a specific set of tags. The list below provides the tags that must be present; in this list, `<cluster-name>` refers to the name of cluster as specified in the `metadata.name` field of the Cluster object (or manifest). Here are the tags that are required:
+Cluster API itself does tag AWS resources it creates. The `sigs.k8s.io/cluster-api-provider-aws/cluster/<cluster-name>` (where `<cluster-name>` matches the `metadata.name` field of the Cluster object) tag, with a value of `owned`, tells Cluster API that it has ownership of the resource and is able to modify the resource and manage the lifecycle of the resource.
 
-    sigs.k8s.io/cluster-api-provider-aws/cluster/<cluster-name>
-    sigs.k8s.io/cluster-api-provider-aws/role
+When consuming existing AWS infrastructure, Cluster API does not require any tags to be present. The absence of the tags on an AWS resource indicates to Cluster API that it should not modify the resource or attempt to manage the lifecycle of the resource.
 
-These two tags should be present on the following AWS resources:
+However, the AWS cloud provider _does_ require certain tags in order to function properly. Specifically, all subnets where Kubernetes nodes reside should have the `kubernetes.io/cluster/<cluster-name>` tag present. Private subnets should also have the `kubernetes.io/role/internal-elb` tag with a value of 1, and public subnets should have the `kubernetes.io/role/elb` tag with a value of 1. These latter two tags help the cloud provider understand which subnets to use when creating load balancers.
```

Comment:
```suggestion
However, the built-in Kubernetes AWS cloud provider _does_ require certain tags in order to function properly. Specifically, all subnets where Kubernetes nodes reside should have the `kubernetes.io/cluster/<cluster-name>` tag present. Private subnets should also have the `kubernetes.io/role/internal-elb` tag with a value of 1, and public subnets should have the `kubernetes.io/role/elb` tag with a value of 1. These latter two tags help the cloud provider understand which subnets to use when creating load balancers.
```

Reply:
I've adopted your change (included in 1667e3ab).